packages/discord.js/package.json

```diff
@@ -56,7 +56,7 @@
     "fast-deep-equal": "^3.1.3",
     "lodash.snakecase": "^4.1.1",
     "tslib": "^2.3.1",
-    "undici": "^4.16.0",
+    "undici": "5.2.0",
```

Comment:
Is there a reason why this doesn't use `^`?

Reply:
idk I just did `yarn workspace @discordjs/rest add undici@5.2.0`. I use npm typically if you couldn't tell

Reply:
NPM would do the same. The command itself was correct, you could also just cd into the packages folder, but you need to add the carret manually if you want to show it up (same behaviour as npm) like: `yarn workspace @discordjs/rest add undici@^5.2.0`

Reply:
npm does it correctly in my experience, even with specificity the caret, yarn doesn't update it properly😕

Reply:
okay so without specifying the version, yarn will add the caret automatically